doc/python/sphinx/glossary.rst

```diff
@@ -27,3 +27,17 @@ Glossary
     the returned object doesn't have restrictions (i.e. ``None`` allowed). The
     deserializer is invoked with inbound message bytes on both the server side
     and the client-side.
+
+  wait_for_ready
+    If an RPC is issued but the channel is in TRANSIENT_FAILURE or SHUTDOWN
+    states, the RPC is unable to be transmitted promptly. By default, gRPC
+    implementations SHOULD fail such RPCs immediately. This is known as "fail
```

Comment:
It's kind of unconventional to describe what implementations *should* do, rather than what *this* implementation actually does. I would get rid of all of the usages of "SHOULD".

I'd recommend that this sentence be changed to "By default, the gRPC library will fail such RPCs immediately."

Reply:
Good idea.